src/transformers/models/falcon_h1/modular_falcon_h1.py

```diff
@@ -617,10 +617,23 @@ def torch_forward(
 
         # 1. Gated MLP's linear projection
         input_states = apply_mask_to_padding_states(input_states, attention_mask)
+        input_states = input_states * self.ssm_in_multiplier   # ADD Mup Multipliers
```

Comment:
```suggestion
        # Add Multipliers
        input_states = input_states * self.ssm_in_multiplier
```
Could you add the same comment in the cuda forward? Keeps it consistent

Reply:
Same for below